packages/node/src/lib/changes/changes.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { Novu } from '../novu';
+import axios from 'axios';
+
+const mockConfig = {
+  apiKey: '1234',
+};
+
+jest.mock('axios');
+
+describe('test use of novus node package - Subscribers class', () => {
+  const mockedAxios = axios as jest.Mocked<typeof axios>;
+  let novu: Novu;
+
+  beforeEach(() => {
+    mockedAxios.create.mockReturnThis();
+    novu = new Novu(mockConfig.apiKey);
+  });
+
+  test('should get changes correctly', async () => {
+    mockedAxios.post.mockResolvedValue({});
+
+    await novu.changes.get();
+
+    expect(mockedAxios.get).toHaveBeenCalled();
+    expect(mockedAxios.get).toHaveBeenCalledWith('/v1/changes');
```

Comment:
here it will be `/changes` instead of `/v1/changes`,  same goes other tests also.